cypress/support/config/envs.js

```diff
@@ -10,6 +10,16 @@ const config = {
     standaloneErrorPages: false,
     alwaysCheckForFallback: true,
   },
+  stage: {
+    baseUrl: 'https://www.stage.bbc.com',
+    dataUrl: 'https://www.stage.bbc.com',
+    assetUrl: 'https://news.stage.files.bbci.co.uk/include/articles/public',
+    assetOrigin: 'https://news.stage.files.bbci.co.uk',
+    atiUrl: 'https://logws1363.ati-host.net?',
+    chartbeatEnabled: true,
+    avEmbedBaseUrl: 'https://www.stage.bbc.com',
+    standaloneErrorPages: false,
```

Comment:
Is there a reason why `alwaysCheckForFallback` was omitted for stage or just an oversight?

Reply:
oversight